### ISSUE TITLE
Diagnostic unit tests: SA1015 (ClosingGenericBracketsMustBeSpacedCorrectly)

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#201]
Implement unit tests for #21.

**Comment by sharwell** [comment]
:exclamation: Make sure to test the following. SA1015 should be reported, and the code fix should remove the space after the comma.

```csharp
var a = typeof(System.Func<, >);
```

**Comment by Noryoko** [comment]
Grabbing this.
